fdmi/plugins/fdmi_sample_plugin.h

```diff
@@ -42,21 +42,14 @@
 #include <fcntl.h>
 #include <stdio.h>
 
-/* sched client conf params */
-struct sched_conf {
-	char          *local_addr;
-	char          *ha_addr;
-	char          *profile_fid;
-	char          *process_fid;
-	char          *fdmi_plugin_fid_s;
-	struct m0_fid  fdmi_plugin_fid;
-} c_params;
+/* Plugin sample conf params */
+struct m0_fsp_params {
```

Comment:
Does `fsp` mean `fdmi sample plugin` ? Please specify in comment here so others can interpret.

Reply:
I think we can define this in the .c file.
No need to have this header file.

Reply:
agreed

Reply:
I agree to move it to the .c file and remove the header file completely.
No need to have such an extra file.

Reply:
done